lib/cloud_controller/logs/request_logs.rb

```diff
@@ -26,11 +26,14 @@ def start_request(request_id, env)
         @incomplete_requests.store(request_id, env)
       end
 
-      def complete_request(request_id, status)
+      def complete_request(request_id, status, env, time_taken)
         return if @incomplete_requests.delete(request_id).nil?
-
-        @logger.info("Completed #{status} vcap-request-id: #{request_id}",
-                     { status_code: status })
+        request = ActionDispatch::Request.new(env)
+        @logger.info("Completed #{status} vcap-request-id: #{request_id}", 
+                     { status_code: status,
+                       time_taken: time_taken,
```

Comment:
What's the unit? Should the name of the field indicate the unit, e.g. `ms`?

Reply:
Updated to ms